### ISSUE TITLE
Packed team format documentation incorrect

### ISSUE THREAD

**Issue by davidstone** [Zarel/Pokemon-Showdown#5145]
https://github.com/Zarel/Pokemon-Showdown/blob/master/PROTOCOL.md#team-format

The example shows HAPPINESS, POKEBALL, and HIDDENPOWERTYPE as being separated by '|', but the format string shows them separated by ','.

**Comment by Zarel** [comment]
> If `POKEBALL` and `HIDDENPOWERTYPE` are both blank, the commas will be left off.

;)

**Comment by Zarel** [comment]
It sounds like you're making a bot, by the way. If so, I'm looking forward to looking at it. Is it open-source somewhere?

**Comment by davidstone** [comment]
Yes: https://bitbucket.org/davidstone/technical-machine and https://www.smogon.com/forums/threads/technical-machine-update.3644813/
